lib/services/staking/reliquary.service.ts

```diff
@@ -270,12 +271,11 @@ export class ReliquaryService {
         const newLevelProgress =
             newLevel > maturityLevels.length ? 'max level reached' : `${newMaturity}/${maturityLevels[newLevel + 1]}`;
 
-        const newLevelProgressDiff = parseInt(maturityLevels[newLevel + 1].toString()) - newMaturity;
-        const oldLevelProgressDiff = parseInt(maturityLevels[levelOnUpdate + 1].toString()) - maturity;
-        const progressDiff = newLevelProgressDiff - oldLevelProgressDiff;
-        const levelDiff = levelOnUpdate - newLevel;
-        const diff = progressDiff + levelDiff * 604800;
-        const diffDate = new Date((nowTimestamp + diff) * 1000);
+        const maxLevel = 10; // zero based indexing, frontend will show '11'
```

Comment:
Should we define this max level in a config somewhere as a constant?

Reply:
Moved it to the config.